NitroxServer/ConsoleCommands/BackCommand.cs

```diff
@@ -15,11 +15,13 @@ protected override void Execute(CallArgs args)
         {
             Validate.IsTrue(args.Sender.HasValue, "This command can't be used by CONSOLE");
             Player player = args.Sender.Value;
+
             if (player.LastStoredPosition == null)
             {
-                Log.InGame("No previous location...");
+                SendMessage(args.Sender, "No previous location...");
```

Comment:
My personal view is this shouldn't be changed as part of this pr as it's irrelevant to the overall theme of the pr

Reply:
I think we can be an exception in this case since it's a one line fix and waiting around for another PR to touch this area is silly.